amazon/ionbenchmark/benchmark_runner.py

```diff
@@ -84,19 +84,23 @@ def run_benchmark(benchmark_spec: BenchmarkSpec):
     raw_timings = timer.repeat(benchmark_spec.get_iterations(), batch_size)
 
     # Normalize the samples (i.e. remove the effect of the batch size) before returning the results
-    nanos_per_op = [t/batch_size for t in raw_timings]
+    nanos_per_op = [t / batch_size for t in raw_timings]
     ops_per_sec = [1000000000.0 / t for t in nanos_per_op]
 
     return BenchmarkResult(nanos_per_op, ops_per_sec, peak_memory_usage)
 
 
-def _create_test_fun(benchmark_spec: BenchmarkSpec):
+def _create_test_fun(benchmark_spec: BenchmarkSpec, return_obj=False, custom_file=False):
     """
     Create a benchmark function for the given `benchmark_spec`.
+
+    :param return_obj: If the test_fun returns the load object for debugging. It only works for `io-type=file` and
```

Comment:
I don't think we're using this pydoc style anywhere else. simpleion uses the google pydoc style. That is the main API to this code, follow what that does.

Reply:
Changed.